src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable react/display-name */
+import React, { memo } from 'react';
+import cn from 'classnames';
+import { Status } from '../../types/Status';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  filter: Status;
+  onFilterChange: (newFilter: Status) => void;
+  todos: Todo[];
+  onDelete: () => void;
+};
+
+export const TodoFilter: React.FC<Props> = memo(
+  ({ filter, onFilterChange, todos, onDelete }) => {
+    const statuses = [
```

Comment:
Move this constant outside the component

Reply:
good point